cdtb/arenadata.py

```diff
@@ -40,9 +41,7 @@ def export(self, output, langs=None):
         """
 
         stringtable_dir = os.path.join(self.input_dir, "data/menu")
-        stringtable_glob = "main_??_??.stringtable"
-        stringtable_format = "main_%s.stringtable"
-        stringtable_regex = r"main_(.._..)\.stringtable$"
+        stringtable_glob, stringtable_format, stringtable_regex = stringtable_strings(stringtable_dir)
```

Comment:
Instead of returning a tuple of those values you can return a list of `(lang, rst_file_path)`.
The method would have to accept `langs` as parameters but it's pretty simple.